rero/rero-ils
# jsonschema: configure the holding editor

The holding JSONSchema is adapted to be usable by the admin ui editor.

* Fixes permissions in the API REST serializer: create is renamed to
cannot_create.
* Fixes the holding importation by defining a default value for the
holding harvested properties.
* Fixes the returned http status when wrong data are posted to the api
holding views.
* Generates documents and persons data after the MEF reindexing.
* Generates holdings and items data.

Co-Authored-by: Johnny Mariéthoz <Johnny.Mariethoz@rero.ch>
Co-Authored-by: Aly Badr <aly.badr@rero.ch>

## Why are you opening this PR?

- Implements: https://tree.taiga.io/project/rero21-reroils/us/1039?milestone=260625

## Dependencies

- rero-ils-ui: https://github.com/rero/rero-ils-ui/pull/216

## How to test?

- run the usual bootstrap and setup commands ( new data has been generated for this PR )
- check holding and item creation for the 3 types of holdings
  - journals: should display a holding editor
  - monography: should display a item editor
  - e-books: no editor should be available
- here is a list of documents with existing serials holdings
  - https://ilsdev.test.rero.ch/professional/records/documents/detail/152
  - https://ilsdev.test.rero.ch/professional/records/documents/detail/1101
  - https://ilsdev.test.rero.ch/professional/records/documents/detail/42
  - https://ilsdev.test.rero.ch/professional/records/documents/detail/617
  - https://ilsdev.test.rero.ch/professional/records/documents/detail/70
  - https://ilsdev.test.rero.ch/professional/records/documents/detail/689
  - https://ilsdev.test.rero.ch/professional/records/documents/detail/1148
  - https://ilsdev.test.rero.ch/professional/records/documents/detail/1089
  - https://ilsdev.test.rero.ch/professional/records/documents/detail/866
  - https://ilsdev.test.rero.ch/professional/records/documents/detail/597
  - https://ilsdev.test.rero.ch/professional/records/documents/detail/1184
- play with the holding editor check the titles and descriptions messages

## Code review check list
